batcher/gnark/verifier.go

```diff
@@ -1,34 +1,55 @@
 package main
 
+/*
+#include <stdlib.h>
+#include <stdint.h>
+
+typedef struct ListRef {
+  const uint8_t *ptr;
+  uintptr_t len;
+} ListRef;
+
+
+*/
+import "C"
```

Comment:
Is it posible to move the import with all the imports?

Reply:
unfortunately no, because of how CGo works, the comment has to ve directly above import.

https://pkg.go.dev/cmd/cgo#hdr-Using_cgo_with_the_go_command